### ISSUE TITLE
Error when adding new item (question/value....) to KM

### ISSUE THREAD

**Issue by thopico** [ds-wizard/ds-wizard#238]
**Describe the bug**
When adding a new item (I first saw it when adding a new response option, then when adding a new question), I hit the following error (see screenshot):
```
The knowledge model entity you are trying to open does not exist.
```

I then need to find my item in the summary and click on it to finally access and customize it. (So after all this bug isn't too constraining).

I upgraded last week from 3.7.0 to 3.8.0. I didn't observe this behavior before. This KM was created with DSW 3.7.0.

**How to reproduce**
Steps to reproduce the problem:
1. Go to Knowledge Model Editor
2. Click on any chapter and try to add a new question OR try to add a new option to any pre-existing question
3. See error

**Expected behavior**
The new item customization page should load

**Screenshots**
![Capture d’écran de 2022-02-07 14-46-03](https://user-images.githubusercontent.com/56619719/152799888-abe56fc1-ba2c-47e6-ba0d-09c4c9cff007.png)

**DS Wizard deployment:**
 - Self hosted dockerized instance (3.8.0)
 - Browser Firefox

Thank you for your help!

**Comment by janslifka** [comment]
I am unable to reproduce the described issue following the provided steps. However, we've released patch version 3.8.1 for `wizard-client` as well as `wizard-server` shortly after 3.8.0 fixing some issues.

Could you please update your instance to the latest version and check if the issue persists?

**Comment by thopico** [comment]
Hi @janslifka ,

sorry I missed this release, but it does fix this behavior with the 3.8.1.

Thank you! I close the issue now.
